flake.nix

```diff
@@ -0,0 +1,125 @@
+          in {
+            # install-method.installMethod = "copy";
+            add-inputs.buildInputs = old: old ++ (lib.attrValues topDependencies) ++ [pkgs.nodejs-14_x.pkgs.yarn];
+            install-symlinks.postInstall = "ln -s $out/lib/node_modules/${n}/dist $out/dist";
+            add-type-notation.prePatch = lib.optionalString (n == "@cardano-sdk/core")
+            ''
+              substituteInPlace ./src/Cardano/util/subtractValueQuantities.ts \
+                --replace "../types"  "@cardano-sdk/core/src/Cardano/types"
+            '';
+            fix-relative-references.postPatch = let
+              references = {
+                "@cardano-sdk/blockfrost" = [ "core" "util" ];
+                "@cardano-sdk/cardano-services-client" = [ "core" "util" ];
+                "@cardano-sdk/cardano-services" = [ "core" "ogmios" "rabbitmq" "util" ];
+                "@cardano-sdk/cip2" = [ "core" "util" ];
+                "@cardano-sdk/cip30" = [ "core" ];
+                "@cardano-sdk/core" = [ "util" ];
+                "@cardano-sdk/e2e" = [ "util" "util-dev" "wallet" "cardano-services" ];
+                "@cardano-sdk/golden-test-generator" = [ "core" "util" ];
+                "@cardano-sdk/ogmios" = [ "core" ];
+                "@cardano-sdk/rabbitmq" = [ "core" "util" ];
+                "@cardano-sdk/util" = [];
+                "@cardano-sdk/util-dev" = [ "core" ];
+                "@cardano-sdk/util-rxjs" = [];
+                "@cardano-sdk/wallet" = [ "cip2" "cip30" "core" "util-rxjs" "util"];
+                "@cardano-sdk/web-extension" = [ "cip30" "core" "wallet" "util-rxjs" "util"];
+              };
+              replacer = lib.concatMapStrings (name: ''
+                --replace \
+                  '"path": "../../${name}/src"' \
+                  '"path": "../node_modules/@cardano-sdk/${name}/src"' \
+              '');
+            in ''
+              cp ${./tsconfig.json} ./src/tsconfig-root.json
+              mkdir build
+              cp ${./build/cjs-package.json} ./build/cjs-package.json
+              cp ${./build/esm-package.json} ./build/esm-package.json
+
+              substituteInPlace ./package.json \
+                --replace '../../build/cjs-package.json' './build/cjs-package.json' \
+                --replace '../../build/esm-package.json' './build/esm-package.json' \
+
+              substituteInPlace ./src/tsconfig.json \
+                --replace \
+                  '"extends": "../../../tsconfig.json"' \
+                  '"extends": "./tsconfig-root.json"' \
+              ${replacer references.${n}}
+
+            '';
+          })
+          subpackages
+        )
+        {
+          "chromedriver" = {
+            dont-download = {
+                buildInputs = old: old ++ [pkgs.chromedriver];
+                CHROMEDRIVER_SKIP_DOWNLOAD = "true";
+            };
+          };
+          # "@cardano-sdk/cardano-services" = {
```

Comment:
Is this going to be used?

Reply:
Maybe (probably) yes, I should remove this for now, though... I haven't yet advanced to the entrypoint packaging, but I'm likely going to need it there.